packages/client/src/workflow-options.ts

```diff
@@ -54,6 +54,12 @@ export interface WorkflowOptions extends CommonWorkflowOptions {
    * @experimental Deployment based versioning is experimental and may change in the future.
    */
   versioningOverride?: VersioningOverride;
+
+  /**
+   * Potentially reduce the latency to start this workflow by encouraging the server to
+   * start it on a local worker running with this same client.
+   */
+  requestEagerStart?: boolean;
```

Comment:
Should it really need to be requested explicitly? Not sure what we did in other languages. If there's really value to this, should consider that as a tri-state? Like `false = absolutely not`, `undefined = if you can, but dont complain`, `true = yes, that's a requirement`.

Reply:
The Python SDK defaults this parameter to `false` for all calls, it doesn't look like it's enabled by default.